src/frontend/js/lib/util/typeahead/lib/Typeahead.ts

```diff
@@ -36,25 +37,28 @@ export class Typeahead {
             source: (query, syncResults, asyncResults) => {
                 if (this.timeout) clearTimeout(this.timeout);
                 this.timeout = setTimeout(() => {
+                    console.log("Typeahead query:", query);
```

Comment:
Could you just remove this debug line please?

Reply:
Apologies - this is now done! :smile: